### PR TITLE
[MCKIN-8184] Update the course enrollment list api endpoint docstring

### PR DESCRIPTION
**Pull request by lgp171188**
Also update the docstring of functions used by the endpoint. This is a follow-up PR to the https://github.com/edx-solutions/edx-platform/pull/1142 PR with fixes to the docstring.

**Reviewers**
- [ ] @pomegranited
- [ ] @xitij2000

### CODE REVIEW

File: common/djangoapps/enrollment/views.py [hunk 1]
```diff
@@ -712,10 +712,12 @@ class CourseEnrollmentsApiListView(DeveloperErrorViewMixin, ListAPIView):
     """
         **Use Cases**
 
-            Get a list of all course enrollments given a course ID or list of usernames.
+            Get a list of all course enrollments, optionally filtered by a course ID or list of usernames.
```
Comment (pomegranited) [inline common/djangoapps/enrollment/views.py]: nit: "optionally filtered by a course ID *and/or* list of usernames"

I know.. programmatically "and/or" is the same as "or", but to humans, I think this is clearer.

File: common/djangoapps/enrollment/views.py [hunk 2]
```diff
@@ -725,15 +727,14 @@ class CourseEnrollmentsApiListView(DeveloperErrorViewMixin, ListAPIView):
...
-              given course ID. The value must be URL encoded. Optional. Required if the 'username' parameter
-              is omitted.
+              given course ID. The value must be URL encoded. Optional.
 
             * username: List of comma-separated usernames. Filters the result to the course enrollments
-              of the given users. Optional. Required if the 'course_id' parameter is omitted.
+              of the given users. Optional.
 
-            * page_size: Number of results to return per page.
+            * page_size: Number of results to return per page. Optional.
```
Comment (pomegranited) [inline common/djangoapps/enrollment/views.py]: Could you indicate the default page size if this parameter is not passed?  Max page size too?

File: common/djangoapps/enrollment/views.py [hunk 3]
```diff
@@ -725,15 +727,14 @@ class CourseEnrollmentsApiListView(DeveloperErrorViewMixin, ListAPIView):
...
 
-            * page: Page number to retrieve.
+            * page: Page number to retrieve. Optional.
```
Comment (pomegranited) [inline common/djangoapps/enrollment/views.py]: Could you indicate the default page number (1)?

**Comment by lgp171188** [comment]
@pomegranited, I have addressed your review comments in https://github.com/edx-solutions/edx-platform/pull/1146/commits/da094934690576b6858ddd6ff93bcf7c862bed87

File: common/djangoapps/enrollment/views.py (hunk 1 above)
Comment (pomegranited) [reply common/djangoapps/enrollment/views.py]: 👍

File: common/djangoapps/enrollment/views.py (hunk 2 above)
Comment (pomegranited) [reply common/djangoapps/enrollment/views.py]: 👍

File: common/djangoapps/enrollment/views.py (hunk 3 above)
Comment (pomegranited) [reply common/djangoapps/enrollment/views.py]: 👍

**Pull request merged**